src/main/webapp/app/exam/participate/exercises/text/text-exam-submission.component.ts

```diff
@@ -97,4 +98,16 @@ export class TextExamSubmissionComponent extends ExamSubmissionComponent impleme
         this.studentSubmission.isSynced = false;
         this.textEditorInput.next((<HTMLTextAreaElement>event.target).value);
     }
+
+    private updateViewFromSubmissionVersion() {
+        if (this.submissionVersion) {
+            if (this.submissionVersion.content) {
+                this.answer = this.submissionVersion.content;
+            }
+        }
+    }
+    setSubmissionVersion(submissionVersion: SubmissionVersion): void {
```

Comment:
```suggestion

    setSubmissionVersion(submissionVersion: SubmissionVersion): void {
```

Reply:
what's your intention here?

Reply:
Just adding a newline before the methods for consistency :)